syncthingwidgets/settings/settingsdialog.cpp

```diff
@@ -809,13 +809,18 @@ bool setAutostartEnabled(bool enabled)
         if (!desktopFile.open(QFile::WriteOnly | QFile::Truncate)) {
             return false;
         }
+#ifndef SYNCTHINGWIDGETS_AUTOSTART_EXEC_PATH
+        // Use the syncthingtray executable located in the directory of the applicationFilePath.
+        // Not using applicationFilePath directly to accomodate for NixOS as well, see:
```

Comment:
```suggestion
        // Not using applicationFilePath directly to accommodate for NixOS as well, see:
```

Reply:
Or maybe just state "for the case of the actual executable being renamed and started via a wrapper" because people reading this that are not involved in NixOS will have no idea what makes NixOS special here and what this code specifically accommodates for.